app/support/notification_sender.rb

```diff
@@ -68,9 +68,9 @@ def reviewed_at
 
   class AccountNotifier
 
-    def notify_accounts(account_ids_to_notify)
+    def notify_accounts(account_ids_to_notify, facility)
```

Comment:
This is another place to check the usage of GlobalID, because I believe this gets delayed.